### ISSUE TITLE
Hi, just want to do one time donation via PayPal $5USD.  Do not see option for you.  Only bunq or pay monthly.

### ISSUE THREAD

**Issue by vhm14u2c** [milo526/homebridge-tuya-web#105]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by milo526** [comment]
Hello,
Thank you for considering a donation.

I have just added a PayPal option

**Comment by vhm14u2c** [comment]
Hi,
Thanks for your reply.
I could not find place you added for PayPal donation.
[cid:D1459D40-570D-4510-B6DE-74E488370D04-L0-001]

For a couple others, I was able to donate via PayPal,
[cid:1F966E10-5E56-4CC5-AC5B-5FE8FCE9A194-L0-001][cid:3439B4C0-5928-4272-8B4F-7695D44A59EE-L0-001]

If you want, I can try to use your email address to send money via PayPal.
Let me know if that’s ok before I do, and thanks.

Vince

Sent from my iPad

On Oct 28, 2020, at 4:25 AM, Milo Cesar <notifications@github.com> wrote:

﻿

Hello,
Thank you for considering a donation.

I have just added a PayPal option
